x-pack/plugins/canvas/public/components/popover/index.ts

```diff
@@ -5,4 +5,4 @@
  * 2.0.
  */
 
-export { Popover, ClosePopoverFn } from './popover';
+export * from './popover';
```

Comment:
Let's leave this as only exporting the specific items we want, not just a `*` so that we make sure we only share what we're intending.

Reply:
Ok, done)